README.md

```
@@ -183,6 +187,13 @@ The Feature Flag Backend project uses DynamoDB as the database. The data model c
 - updatedAt (number)
 - updatedBy (string)
 
+### requestLimit
+- limitType (string) (partitionKey) 
+- limitValue (number)
+
+## Deployment instructions 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)